packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -144,28 +144,41 @@ enum ModelFramework {
 const isProcessing = ref(false);
 
 const editor = ref<VAceEditorInstance['_editor'] | null>(null);
-const modelName = ref('');
```

Comment:
removing the modelname for now - I will handle this in a later PR.  But the flow to name a model is a little confusing at the moment